test/type/number_test.exs

```diff
@@ -0,0 +1,131 @@
+defmodule Litmus.Type.NumberTest do
+  use ExUnit.Case
+
+  alias Litmus.Type
+
+  describe "validate_field/3" do
+    test "validates property values of data based on their Number schema definition in Type.Number module" do
```

Comment:
this is a rather verbose test description, can we make it more concise? Perhaps `validates Type.Number fields in a schema`?